app/graph/types/mutation_type.rb

```diff
@@ -128,6 +128,10 @@
   field :updateFactCheck, field: FactCheckMutations::Update.field
   field :destroyFactCheck, field: FactCheckMutations::Destroy.field
 
+  field :createFeed, field: FeedMutations::Create.field
+  field :updateFeed, field: FeedMutations::Update.field
+  field :destroyFeed, field: FeedMutations::Destroy.field
```

Comment:
Suggestion: We won't use `destroyFeed` for now, so no need to expose it.

Reply:
Done